templates/components/common/header.html

```diff
@@ -24,8 +24,8 @@ <h1 class="header-logo header-logo--{{theme_settings.logo-position}}">
             {{> components/common/store-logo}}
         </div>
     {{/if}}
-    {{{region name="header_bottom"}}}
     <div class="navPages-container" id="menu" data-menu>
         {{> components/common/navigation-menu}}
     </div>
 </header>
+{{{region name="header_bottom"}}}
```

Comment:
If this is header bottom shouldn't this be inside the header tag ?

Reply:
So i spoke with @pedelman / @BC-EChristensen and i believe they wanted to move this outside of the header tag due to its placement. I can move this back if necessary, but they may be able to speak on this as well!

Reply:
I don't want you to move ... I am just trying to understand the naming and the reasoning behind it.

Reply:
Unfortunately we already have two regions, `header_bottom` and `product_below_price`, using two different naming schemes. We opted to adopt the second, but unfortunately we can't change this one.

You're right about `menu_bottom` below, though, we're changing that to `home_below_menu`